### PR TITLE
Add video latency description

### LINKED ISSUE #2109: Add video latency constrainable property
Can we extended existing [latency](https://w3c.github.io/webrtc-pc/#mediatracksupportedconstraints-mediatrackcapabilities-mediatrackconstraints-and-mediatracksettings) constrainable property API to support video MediaStreamTrack sourced by an RTCPeerConnection? This extensions would open a room for quality improvements for video MediaStreamTrack sourced by an RTCPeerConnection as you will have additional time until the video playout. For example, in the case of network issues you will have more video samples to play and recover. Or if the packets are lost then you will have more time for packets retransmission. Another problem it might solve is audio video desynchronization. Currently you have option to only increase latency for the audio track and if it has associated video track then they would be desynced.

### PR DESCRIPTION
**Pull request by kuddai**
Fixes #2109

***
<a href="https://pr-preview.s3.amazonaws.com/kuddai/webrtc-pc/pull/2110.html" title="Last updated on Mar 22, 2019, 10:45 AM UTC (aabe719)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2110/8135bd9...kuddai:aabe719.html" title="Last updated on Mar 22, 2019, 10:45 AM UTC (aabe719)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -10932,9 +10932,11 @@ <h4>MediaTrackSupportedConstraints, MediaTrackCapabilities,
...
         <code>setRemoteDescription</code> and the actual RTP data. This means
         that certain members, such as <code>facingMode</code>,
-        <code>echoCancellation</code>, <code>latency</code>,
-        <code>deviceId</code> and <code>groupId</code>, will always be
-        missing.</p>
+        <code>echoCancellation</code>, <code>deviceId</code> and <code>groupId
+        </code>, will always be missing.</p>
+        <p>The <code>latency</code> setting on the <code>MediaStreamTrack</code>
+        sourced by an <code>RTCPeerConnection</code> describes the delay between
+        receiving a media packet and rendering that media.
```
Comment (henbos) [inline webrtc.html]: "describes the delay introduced by the RTCPeerConnection implementation (such as a jitter buffer delay)."

Clarify that this is for both audio and video (in mediacapture-main this is audio-only)

File: webrtc.html [hunk 2]
```diff
@@ -10932,9 +10932,12 @@ <h4>MediaTrackSupportedConstraints, MediaTrackCapabilities,
         <code><a>RTCSessionDescription</a></code> applied via
         <code>setRemoteDescription</code> and the actual RTP data. This means
         that certain members, such as <code>facingMode</code>,
-        <code>echoCancellation</code>, <code>latency</code>,
-        <code>deviceId</code> and <code>groupId</code>, will always be
-        missing.</p>
+        <code>echoCancellation</code>, <code>deviceId</code> and <code>groupId
+        </code>, will always be missing.</p>
+        <p>The <code>latency</code> setting on the audio or video <code>
```
Comment (henbos) [inline webrtc.html]: Refer to it as a "constrainable property". A setting is a specific value at a particular point in time, not the thing that you are constraining.

**Comment by henbos** [comment]
Do you know why ipr still is not green?

**Comment by kuddai** [comment]
> Do you know why ipr still is not green?

I did as you recommended, first linked my github and W3C account, then I made another commit. But judging by email I have received Today this integration takes some time to propagate.

**Comment by alvestrand** [comment]
IPR is green now.

**Comment by alvestrand** [comment]
Unclear: Is the request for a read-only property or the ability to try to constrain the property?
These two have different costs in terms of how complicated it is to implement.

**Comment by henbos** [comment]
@jan-ivar to comment

**Comment by kuddai** [comment]
It is

> Unclear: Is the request for a read-only property or the ability to try to constrain the property?
> These two have different costs in terms of how complicated it is to implement.

It is the ability to try to constrain the latency property.

**Comment by henbos** [comment]
Ping @jan-ivar

**Comment by aboba** [comment]
@henbos to create slide(s) for the virtual interim.

**Comment by henbos** [comment]
Jitter buffer API has moved to the receiver, which will move to an extension spec. Still need to clarify constraints but that is a separate issue.

**Pull request closed without merging**